src/components/header/examples/header.tsx

```diff
@@ -0,0 +1,48 @@
+import { Component, h, State } from '@stencil/core';
+import { Action } from '@limetech/lime-elements';
+
+@Component({
+    tag: 'limel-example-header',
+    shadow: true,
+})
+export class HeaderExample {
+    @State()
```

Comment:
I am not sure if it is a `State` here

Reply:
No, since they don't change, the State decorator isn't necessary, and just wires up unnecessary overhead.

Reply:
Fixed!